portal/models/i18n.py

```diff
@@ -16,13 +16,19 @@
 from .coding import Coding
 from .intervention import Intervention
 from .organization import Organization
+from .overall_status import OverallStatus
 from .questionnaire_bank import QuestionnaireBank, classification_types_enum
 from .research_protocol import ResearchProtocol
 from .role import Role
 from .user import current_user
 
 
 def get_db_strings():
+    """
```

Comment:
As suggested by pep8, we tend to use the [Multi-Line Docstring](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) format - so that first summary line is just right of the opening triple quotes.